diagnostic.md

```diff
@@ -4,7 +4,10 @@ Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
-
+class
+id
+role
+div
```

Comment:
I don't think you fully understand what is meant by "selector." Take a look at this article: http://www.w3schools.com/jquery/jquery_ref_selectors.asp